topics/data-science/tutorials/gnmx-lecture1/tutorial.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: tutorial_hands_on
+
+title: "A (very) brief history of genomics"
+questions:
+  - What is this course about?
+  - What is the history of genomics?
+objectives:
+  - Have a basic understanding of history of biology from Darwin to today. 
+time_estimation: "1h"
+key_points:
+  - Jim Watson did not discover DNA
+contributions:
+  authorship:
+  - nekrut
+
+subtopic: gnmx
+draft: true
+---
+
+![Fluctuation test](./images/luria_small.png)
+
+# What do you know?
+
+This is the first lecture of the class. We will begin with a short quiz that is designed tio let me know what you do and do not know. This quiz will have no effect on your grade in this course. 
```

Comment:
would you be willing to share the quiz? we have a built in quiz feature but it's probably not ready for prime-time, but we might be able to make use of it later.

Reply:
sure. once I make it ;)